lib/devise/models/confirmable.rb

```diff
@@ -117,8 +117,11 @@ def send_confirmation_instructions
           generate_confirmation_token!
         end
 
-        opts = pending_reconfirmation? ? { to: unconfirmed_email } : { }
-        send_devise_notification(:confirmation_instructions, @raw_confirmation_token, opts)
+        if pending_reconfirmation?
```

Comment:
What do you think about change this `if`to:
```ruby
params = {}
params[:to] = unconformed_email if pending_reconfirmation?
send_devise_notification(:confirmation_instructions, @raw_confirmation_token, params)
```

Reply:
Thank you for your review.
I think `send_devise_notification` 's first argument must use each for calling `Devise::Mailer#confirmation_instructions, reconfirmation_instructions`